src/Tasks/ManifestUtil/ApplicationManifest.cs

```diff
@@ -255,13 +262,15 @@ public string OSSupportUrl
         }
 
         /// <summary>
-        /// Specifies the minimum required OS version required by the application.
-        /// An example value is "5.1.2600.0" for Windows XP.
-        /// If this input is not specified a default value is used.
-        /// The default value is the minimum supported OS of the .NET Framework, which is "4.10.0.0" for Windows 98SE.
-        /// However, if the application contains any native or Reg-Free COM references, then the default will be the Windows XP version.
-        /// For a Win32 native manifest, this input is ignored.
+        /// Gets or sets the minimum OS version required by the application.
         /// </summary>
+        /// <remarks>
+        /// An example value is "5.1.2600.0" for Windows XP.        
+        /// If you don't specify a value, a default value is used.
+        /// The default value is the minimum supported OS of the .NET Framework, which is "4.10.0.0" for Windows 98 Second Edition.
```

Comment:
@danmoseley Should this MSBuild logic be updated since Windows 98 is no longer supported?

Reply:
This isn't my code/product. I believe this relates to ClickOnce, which I don't think (?) is under active development. So my inclination if I owned this would be to not touch functionality unless I have to. I would just update the comment as seems best. It doesn't seem very consequential if it's not quite right.

Reply:
Team triage: We've seen some PRs from ClickOnce, so it still seems to be under active development. @sujitnayak and @John-Hart can comment on whether they want to make a change, though I agree it doesn't seem too consequential.

Reply:
ClickOnce is still supported and in fact, we have updated this recently for .NET Core, but as to which versions of Windows and .NET that are supported or the defaults is more of a ClickOnce Runtime question for either @merriemcgaw  or @NikolaMilosavljevic

Reply:
I stand corrected.

Reply:
@merriemcgaw, @NikolaMilosavljevic, update?

Reply:
I don't think we should modify the default supported OS. Let's keep the old value intact. Customers can always modify this on their own.